apps/native-component-list/src/screens/Image/ImageImperativeApiScreen.tsx

```diff
@@ -0,0 +1,90 @@
+import { Image } from 'expo-image';
+import React from 'react';
+import { Button, ScrollView, StyleSheet, View } from 'react-native';
+import Animated, {
+  useAnimatedRef,
+  useAnimatedStyle,
+  useSharedValue,
+  withRepeat,
+  withSequence,
+  withTiming,
+} from 'react-native-reanimated';
+
+const REAImage = Animated.createAnimatedComponent(Image);
+const REAView = Animated.createAnimatedComponent(View);
+
+export default function ImageImperativeApiScreen() {
+  const value = useSharedValue(50);
+  const ref = useAnimatedRef<Image>();
+
+  const animatedStyle = useAnimatedStyle(() => ({
+    width: value.value,
+    height: value.value,
+  }));
+
+  return (
+    <ScrollView style={styles.container}>
+      <Button
+        title="animate to 500"
+        onPress={() => {
+          value.value = withTiming(500, { duration: 500 });
+        }}
+      />
+      <Button
+        title="animate to 50"
+        onPress={() => {
+          value.value = withTiming(50, { duration: 1000 });
+        }}
+      />
+      <Button
+        title="animate"
+        onPress={() => {
+          value.value = withRepeat(
+            withSequence(withTiming(50, { duration: 500 }), withTiming(500, { duration: 500 })),
+            2,
+            true
+          );
+        }}
+      />
+      <Button
+        title="lock resource"
```

Comment:
Can you change that title to make it more descriptive? Someone during QA may not know what this function is supposed to do.